runtime/sema/errors.go

```diff
@@ -1012,6 +1012,34 @@ func (e *DuplicateConformanceError) Error() string {
 
 func (*DuplicateConformanceError) isSemanticError() {}
 
+// MultipleInterfaceDefaultImplementationsError
+
+// TODO: just make this a warning?
+
```

Comment:
I think it makes sense to have this be an error for now, i.e. be more conservative

```suggestion
//
```